at-a-glance.rst

```diff
@@ -1,43 +1,41 @@
-`Goutte`_. Those browser emulators send a real HTTP requests against an application
+The first type of browser emulators are simple pure HTTP specification implementations, like
+`Goutte`_. These browser emulators run real HTTP requests against an application
 and parse the response content. They are very simple to run and configure,
-because this type of emulators can be written in any available programming
-language and can be run through console on servers without GUI. Headless
-emulators have both advantages and disadvantages. Advantages are simplicity,
-speed and ability to run it without the need of a real browser. But this
-type of browsers has one big disadvantage, they have no JS/AJAX support.
-So, you can't test your rich GUI web applications with headless browsers.
+because these type of emulators can be written in any available programming
+language and can be run via the command line on servers without GUI. Headless
+emulators have both advantages and disadvantages. The advantages are simplicity,
+speed and the ability to run it without the need of a real browser. These
+types of emulator have one big disadvantage in that they have no JS/AJAX support.
+This means that you can't test your rich GUI web applications with headless browsers.
 
-Second browser emulators type are browser controllers. Those emulators aims
-to control the real browser. That's right, a program to control another program.
-Browser controllers simulate user interactions on browser and are able to
+The second type of browser emulators are browser controllers. These emulators aim
+to manipulate the real browser. That's right, a program to control another program.
+Browser controllers simulate user interactions on a browser and are able to
 retrieve actual information from current browser page. `Selenium`_ and `Sahi`_
 are the two most famous browser controllers. The main advantage of browser
-controllers usage is the support for JS/AJAX interactions on page. Disadvantage
-is that such browser emulators require the installed browser, extra configuration
-and are usually much slower than headless counterparts.
+controller usage is the support for JS/AJAX interactions on a page. The disadvantages
+are that such browser emulators require the installed browser, extra configuration
+and are usually much slower than their headless counterparts.
 
 So, the easy answer is to choose the best emulator for your project and use
 its API for testing. But as we've already seen, both browser emulator types have both
-advantages and disadvantages. If you choose headless browser emulator, you
-will not be able to test your JS/AJAX pages. And if you choose browser controller,
-your overall test suite will become very slow at some point. So, in real
+advantages and disadvantages. If you choose headless browser emulators, you
+will not be able to test your JS/AJAX pages, and if you choose browser controllers then
```

Comment:
This sentence is actually wrong. Zombie.js is a headless browser emulator supporting JS